tests/NovelRT.Tests/Interop/Maths/QuadTreeTest.cpp

```diff
@@ -290,16 +279,6 @@ TEST_F(InteropQuadTreeTest, insertFiveDoesSubdivideAndPointsAreCorrect)
     EXPECT_EQ(outputChildTreeSizeBottomLeft, 1u);
     EXPECT_EQ(outputChildTreeSizeBottomRight, 1u);
 
-    NrtQuadTreeHandle treeOutputTwoTopLeft = nullptr;
-    NrtQuadTreeHandle treeOutputTwoTopRight = nullptr;
-    NrtQuadTreeHandle treeOutputTwoBottomLeft = nullptr;
-    NrtQuadTreeHandle treeOutputTwoBottomRight = nullptr;
-
-    int32_t treeResultTwoTopLeft = Nrt_QuadTree_getTopLeft(_quadTree, &treeOutputTwoTopLeft);
-    int32_t treeResultTwoTopRight = Nrt_QuadTree_getTopRight(_quadTree, &treeOutputTopRight);
-    int32_t treeResultTwoBottomLeft = Nrt_QuadTree_getBottomLeft(_quadTree, &treeOutputBottomLeft);
-    int32_t treeResultTwoBottomRight = Nrt_QuadTree_getBottomRight(_quadTree, &treeOutputBottomRight);
-
```

Comment:
Mobile makes this a pain to read - what actually happened here?

Reply:
All these were not used at all in the tests. I can add tests if needed but it will require a bit more time. :)

Reply:
If they weren't used then nice spot + cleanup. The less code we have to maintain the better.